scripts/messages/withdrawTokenSimple.ts

```diff
@@ -0,0 +1,55 @@
+import { ethers } from 'hardhat';
+import { getPersonalWallet } from '../utils/wallet';
+
+async function main() {
+    const signer = getPersonalWallet(ethers.provider);
+    const bridgeAddress = process.env.BRIDGE_ADDRESS!;
+    const tokenAddress = process.env.TOKEN_ADDRESS!;
+    const recipient = process.env.RECIPIENT!;
+    const amount = process.env.AMOUNT!;
+
+    if (!bridgeAddress || !tokenAddress || !recipient || !amount) {
+        throw new Error('BRIDGE_ADDRESS, TOKEN_ADDRESS, RECIPIENT, and AMOUNT env vars required');
+    }
+
+    const bridge = await ethers.getContractAt('TestBridge', bridgeAddress, signer);
+    const token = await ethers.getContractAt('IERC20', tokenAddress, signer);
+
+    // Get token info
+    let tokenDecimals = 18;
+    try {
+        const erc20Detailed = await ethers.getContractAt("TestToken", tokenAddress);
+        tokenDecimals = await erc20Detailed.decimals();
+    } catch {
+        console.log('Could not get token decimals, using 18');
+    }
+
+    // Get the fee from the blockchain configuration
+    const tokenBridge = await bridge.tokenBridges(tokenAddress);
+    const feeAmount = tokenBridge.config.fee;
+
+    console.log(`Using fee from blockchain: ${ethers.formatEther(feeAmount)} ETH`);
+
+    const withdrawalAmount = ethers.parseUnits(amount, tokenDecimals);
```

Comment:
chore: same here with the amount divisible check based on the `config.decimalScalingFactor`.